webapp-django/crashstats/signature/templates/signature/signature_summary.html

```diff
@@ -1,6 +1,7 @@
-<div class="panel">
-    <header class="title">
+<div class="panel tab-inner-panel">
```

Comment:
It's because the associated CSS styles are for the ``tab-inner-panel`` context, so doing this allows me to avoid changing the CSS too much, and potentially messing with other parts of the code.